site/cypress/e2e/Cypress-Gradeable/grading_stats.spec.js

```diff
@@ -16,79 +16,99 @@ describe('Test cases for grading stats', () => {
             cy.login(user);
         });
 
-        it(`${user} view should be accurate for teams.`, () => {
+        it(`${user} view should be accurate for team gradeables.`, () => {
             //Team gradeables on Sample Course don't have teams in Null section
             cy.visit(['sample', 'gradeable', 'grading_team_homework', 'grading', 'status']);
-            cy.get('#left-grading-stats').as('on_time_submissions');
-            cy.get('@on_time_submissions').should('contain', 'Students on a team: 101/101 (100%)');
-            cy.get('@on_time_submissions').should('contain', 'Number of teams: 36');
-            cy.get('@on_time_submissions').should('contain', 'Teams who have submitted on time: 25 / 36 (69.4%)');
-            cy.get('@on_time_submissions').should('contain', 'Section 1: 1 / 3 (33.3%)');
+            cy.get('#left-grading-stats').as('left-chunk-stats');
+            cy.get('#right-grading-stats').as('right-chunk-stats');
+            cy.get('#grader-info').as('right-chunk-grader-info');
+            cy.get('@left-chunk-stats').should('contain', 'Students on a team: 101/101 (100%)');
+            cy.get('@left-chunk-stats').should('contain', 'Number of teams: 36');
+            cy.get('@left-chunk-stats').should('contain', 'Teams who have submitted on time: 25 / 36 (69.4%)');
+            cy.get('@left-chunk-stats').should('contain', 'Section 1: 1 / 3 (33.3%)');
+            cy.get('@right-chunk-stats').should('contain', 'Average: 9.00 / 12 (75%)');
+            cy.get('@right-chunk-grader-info').should('contain', 'instructor (count 10 - avg 1.15 - stddev 0.95)');
```

Comment:
Maybe variable names could be more descriptive like the original code? Like instead of left-chunk-stats and right-chunk-stats copy-pasted, it could be on_time_submissions_left, on_time_submissions_right, all_submissions_left, all_submissions_right, etc.